upcoming-release-notes/5761.md

```diff
@@ -0,0 +1,7 @@
+---
+category: Bugfix
+authors: [thromer]
+---
+Avoid repeated calls to payees.find. Fixes #5755.
+
```

Comment:
It would be good to make this more understandable to people reading the release notes.

```suggestion
Optimise the way payee information is fetched.
```